students/BrianNisonger/break_me.py

```diff
@@ -0,0 +1,18 @@
+def NameError():
+	print (x)
+
+def TypeError():
+	a=5;
```

Comment:
what the semi-colon doing there? it will be ignored, but no reason to put it there.